scientific_library/tvb/basic/neotraits/matjax.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
```

Comment:
Does this new file really fit under `neotraits` ? 
From a logical perspective it does not seem fitting here to me. Is it ?
Not sure where to put it, but maybe closer to where it is used (in the web interface e.g. `framework_tvb/tvb/interfaces/web/controllers/burst/`) ?

Reply:
Okay, I moved it there.